Update service-fabric-cluster-security-update-certs-azure.md

aljo-microsoft opened the pull request:
Updating documentation to articulate the change in default certificate load behavior.

azurecla commented:
Hi __@4c74356b41__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

PRmerger commented:
@4c74356b41 : Thanks for your contribution to the Azure documentation! The author, @narayanannamalai, has been notified to review your proposed change.

NarayanAnnamalai commented:
#merge-as-is  
Ready to merge. #sign-off

PRmerger commented:
Only the author of the article can use the sign off comment to add the ready-to-merge label.

NarayanAnnamalai commented:
#merge as-is

TerryLanfear commented:
@NarayanAnnamalai - 4c74356b41 let me know that there were PRs that seemed to be stalled.  Do you know who your pull request approver is?  Can you message your PR approver to merge?

4c74356b41 commented:
@TerryLanfear sorry to derail this thread, yesterday about 10 of my pr's got approved to merge (most of them by you, thanks!), but still some are kinda old with no activity. Also there are some that you are aware of (with this same person from this thread), but they are not really old, like 2-4 days, thats pretty normal. Btw, he already approved a bunch of my PR's so I'm not worried about those, just telling the whole picrute.

All of the old ones (that appear stalled):
https://github.com/Azure/azure-content/pull/7021
https://github.com/Azure/azure-content/pull/7090
https://github.com/Azure/azure-content/pull/7091
https://github.com/Azure/azure-content/pull/7095

Also there are like 5 of PR's that should be approved by @azdocprs with no signs of activity. All 3 days old (so perhaps not stalled).
And again, thanks for helping me. I don't think my PR's are something that can change the world (or Azure documentation), but you have to start somewherre ;)

jomolnar commented:
@NarayanAnnamalai Hi Narayan. Please resolve the conflicts to enable a merge. Usually, authors of the files are able to fix merge conflicts using the process here on the [wiki](https://microsoft.sharepoint.com/teams/azurecontentguidance/wiki/Pages/Resolve a local merge conflict yourself.aspx). Thanks.

4c74356b41 commented:
Just a ping ;)

jimdial commented:
@4c74356b41 . Apologies for the delayed reply. I believe all of your edits were incorporated into the article outside of this PR. If you believe there are still errors, please file a separate PR.

Thank you! #please-close

aljo-microsoft committed:
Update service-fabric-cluster-security-update-certs-azure.md

Updating documentation to articulate the change in default certificate load behavior.
